Libraries/Hotcakes.Shipping.UpsFreight/ServiceTypes.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
```

Comment:
This is missing the open source license header

Reply:
done,added license header.